ooipy/request/hydrophone_request.py

```diff
@@ -512,7 +517,9 @@ def __map_concurrency(func, iterator, args=(), max_workers=-1):
     with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers) as executor:
         # Start the load operations and mark each future with its URL
         future_to_url = {executor.submit(func, i, *args): i for i in iterator}
-        for future in concurrent.futures.as_completed(future_to_url):
+        for future in tqdm(
+            concurrent.futures.as_completed(future_to_url), total=len(iterator), disable=not verbose
+        ):
```

Comment:
```suggestion
        # Disable progress bar
        is_disabled = not verbose
        for future in tqdm(
            concurrent.futures.as_completed(future_to_url), total=len(iterator), disable=is_disabled
        ):
```

Just a little tweak for code readability.. I wasn't sure what the verbose thing was first time I read it.. I think this way would make it more clear